src/test/resources/application.properties

```diff
@@ -1,9 +1,15 @@
 spring.application.name=EasyStay
 
+server.servlet.context-path=/api
+
 spring.datasource.url=jdbc:h2:mem:testdb
 spring.datasource.driverClassName=org.h2.Driver
 spring.datasource.username=sa
 spring.datasource.password=password
-stripe.api.key=invalid_Stripe_api_key
 
 spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
+
+stripe.api.key=invalid_Stripe_api_key
+
+BOT_USERNAME=rent_masters_test_bot
+BOT_TOKEN=7472526149:AAGaNEm5eISWTER61Vnyi-K8mozKlStXVEI
```

Comment:
Is it ok to push it to gitHub?

Reply:
Yes, it is bot just for tests. Without it, the app didn't start. I use another bot for app.